presto-orc/src/main/java/com/facebook/presto/orc/OrcOutputBuffer.java

```diff
@@ -386,12 +387,14 @@ public String toString()
 
     private void ensureWritableBytes(int minWritableBytes)
     {
+        assert minWritableBytes <= maxBufferSize;
```

Comment:
We don't use `assert` in the codebase. Replace it with `checkState(...)` or `verify(...)` for state check or assertion. We also use  `checkArgument` to check arguments.